package.json

```diff
@@ -11,11 +11,16 @@
     },
     "devDependencies": {
         "axios": "^0.19",
+        "bootstrap": "^4.4.1",
         "cross-env": "^5.1",
+        "jquery": "^3.5.1",
         "laravel-mix": "^4.0.7",
+        "laravel-mix-purgecss": "^5.0.0",
         "lodash": "^4.17.13",
+        "popper.js": "^1.16.1",
         "resolve-url-loader": "^2.3.1",
         "sass": "^1.15.2",
-        "sass-loader": "^7.1.0"
+        "sass-loader": "^7.1.0",
+        "vue-template-compiler": "^2.6.11"
```

Comment:
Could you link the pages for these dependencies so the function of them can be reviewed?

Reply:
Sure:

* [Bootstrap](https://getbootstrap.com), [JQuery](https://jquery.com/) and [Popper.js](https://popper.js.org/) were already included in the template via a CDN and are required for functionality of Bootstrap's interactive components
* [laravel-mix-purgecss](https://github.com/spatie/laravel-mix-purgecss) is used to use PurgeCSS with Laravel Mix
* [vue-template-compiler](https://www.npmjs.com/package/vue-template-compiler) is a package by [Vue.js](https://vuejs.org/)'s team used to compile Vue files into JS. It's not needed in this project but Laravel Mix is designed in a way that it's unfortunately needed when using Mix with JS files.[[1]](https://github.com/JeffreyWay/laravel-mix/issues/1891)